snooty/postprocess.py

```diff
@@ -1651,23 +1651,43 @@ def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
         if not isinstance(node, n.Directive) or node.name != "facet":
             return
 
+        def get_children_total(
+            child: n.Node,
+        ) -> int:
+            if not isinstance(node, n.Directive) or node.name != "facet":
```

Comment:
NIT: Since we check if the node is a directive or not right before the function is declared, we can probably omit this check here.

Reply:
good catch! remnant of excessive checking